connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceConnectorTest.java

```diff
@@ -261,6 +328,114 @@ public void testNewTopicConfigs() throws Exception {
         verify(connector).createNewTopics(any(), any());
     }
 
+    @Test
+    @Deprecated
+    public void testIncrementalAlterConfigsRequested() throws Exception {
+        Map<String, String> props = makeProps();
+        props.put(MirrorSourceConfig.USE_INCREMENTAL_ALTER_CONFIGS, MirrorSourceConfig.REQUEST_INCREMENTAL_ALTER_CONFIGS);
+        MirrorSourceConfig connectorConfig = new MirrorSourceConfig(props);
+
+        MockAdminClient admin = spy(new MockAdminClient());
```

Comment:
We don't actually have to use the `MockAdminClient` class at all and can just mock the `Admin` interface for this test:
```suggestion
        Admin admin = mock(Admin.class);
```